webapp/app.py

```diff
@@ -245,7 +245,6 @@ def contribute_index():
         session=session,
         site="vanillaframework.io/docs",
         template_path="docs/search.html",
-        search_engine_id="adb2397a224a1fe55",  # https://cse.google.co.uk/cse/setup/basic?cx=adb2397a224a1fe55
```

Comment:
With the new version, you can have this `search_engine_id` or not have it, but it's best to remove it until UX + Marketing reach a decision on how we want to separate searches.